src/aws/nodeworker/nodeworker.py

```diff
@@ -53,15 +56,28 @@ async def process(self):
             while True:
                 if not self.current_task and self._task_queue:
                     self.current_task = self._task_queue.pop(0)
+
                     log_info("Downloading File " + self.current_task.file_path + ".")
                     file = self.storage_connector.download_file(
                         file_path=self.current_task.file_path,
                         key=self.current_task.key
                     )
                     log_info("Downloaded file " + self.current_task.file_path + ".")
+                    # file = self.storage_connector.download_file(
+                    #     file_path=self.current_task.file_path,
+                    #     key=self.current_task.key
+                    # )
+                    input_data = self.current_task(0).get_task_data()
+                    input_sequences = Tokenize.tokenize_text(
+                        os.path.join("models", "tokenizer_20000.pickle"), input_data)
+                    model = load_model(os.path.join("models", "Senti.h5"))
+                    labels = model.predict(input_sequences)
```

Comment:
What is the result of the model? Is the result labels?

Reply:
The result is a binary np array 1x6 i.e. [0,1,0,1,1,1]. This maps to the sentiments. Should this value be returned?

Reply:
We need to know the precision or accuracy or some metric to know how good the model is. This was said in the experiment. We could also choose to remove this from the paper but it makes our story a bit more complete